lib/gooddata/lcm/lcm2.rb

```diff
@@ -365,6 +365,8 @@ def perform(mode, params = {})
           end
         end
 
+        GoodData::Bricks::ExecutionResultMiddleware.update_execution_result
```

Comment:
should called with explicit status: update_execution_result(GoodData::Bricks::ExecutionStatus::OK)

Reply:
Yes, I updated all comment.
Now , I will reopen release ruby PR branch to verify again